extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -23,6 +23,26 @@ import {toggle} from '../../../src/style';
 
 
 class AmpStickyAd extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {!Element} */
```

Comment:
You want `scheduleLayout(dev().assertElement(this.ad_))` if you are sure that it is no longer `null`